### PR TITLE
Fix performance tests

### PR DESCRIPTION
**Pull request by fabricio-aguiar**
[noissue]

### CODE REVIEW

File: templates/travis/.travis/performance_test_script.sh.j2 [hunk 1]
```diff
@@ -1,5 +1,17 @@
 #!/bin/bash
 
-if [ "$TRAVIS_EVENT_TYPE" = "cron" ]; then
+{% include 'header.j2' %}
+
+set -euv
+
+django-admin runserver 24817 >> ~/django_runserver.log 2>&1 &
```
Comment (dkliban) [inline templates/travis/.travis/performance_test_script.sh.j2]: This won't work because pulp-api is in a container.

Comment (fabricio-aguiar) [reply templates/travis/.travis/performance_test_script.sh.j2]: I copied it from here: https://github.com/pulp/plugin_template/blob/master/templates/travis/.travis/publish_client_pypi.sh.j2#L9

File: templates/travis/.travis/script.sh.j2 [hunk 2]
```diff
@@ -114,6 +114,12 @@ export PYTHONPATH=$TRAVIS_BUILD_DIR:${PYTHONPATH}
 export PYTHONPATH=$TRAVIS_BUILD_DIR:$TRAVIS_BUILD_DIR/../pulpcore:${PYTHONPATH}
 {% endif %}
 set -u
+
+if [[ "$TEST" == "performance" ]]; then
+  echo "--- Performance Tests ---"
+  pytest -vv -r sx --color=yes --pyargs --durations=0 {{ plugin_snake }}.tests.performance || show_logs_and_return_non_zero
+  exit
+
```
Comment (daviddavis) [inline templates/travis/.travis/script.sh.j2]:
```suggestion
fi

```

Comment (fabricio-aguiar) [reply templates/travis/.travis/script.sh.j2]: thanks @daviddavis

**Pull request merged**